qiskit_optimization/algorithms/optimization_algorithm.py

```diff
@@ -99,12 +100,12 @@ class OptimizationResult:
 
     def __init__(
         self,
-        x: Optional[Union[List[float], np.ndarray]],
-        fval: Optional[float],
+        x: Union[List[float], np.ndarray] | None,
+        fval: float | None,
         variables: List[Variable],
         status: OptimizationResultStatus,
-        raw_results: Optional[Any] = None,
-        samples: Optional[List[SolutionSample]] = None,
+        raw_results: Any | None,
+        samples: List[SolutionSample] | None,
```

Comment:
These seem to have lost the `= None` default value

Reply:
oops, thank you!